pkgs/top-level/all-packages.nix

```diff
@@ -6578,6 +6597,16 @@ in
   boost155 = callPackage ../development/libraries/boost/1.55.nix { };
   boost159 = callPackage ../development/libraries/boost/1.59.nix { };
   boost160 = callPackage ../development/libraries/boost/1.60.nix { };
+  boost160libcxx38 = callPackage ../development/libraries/boost/1.60.nix {
```

Comment:
I don't think most people will need this, as it's an irods implementation detail. I'd put this expression inside your irods directory (and probably depend on `boost160` in your packages, then call `boost160.override { stdenv = ...; libiconv = ...; }`)